x-pack/legacy/plugins/maps/public/layers/styles/vector/components/color/dynamic_color_form.js

```diff
@@ -95,7 +95,7 @@ export function DynamicColorForm({
         <EuiFlexItem>
           <FieldSelect
             fields={fields}
-            selectedFieldName={_.get(styleOptions, 'field.name')}
+            selectedFieldName={styleProperty.getField() ? styleProperty.getField().getName() : ''}
```

Comment:
this is not relevant to the bug, but changed this for consistency.

Reply:
All of the style property form components are pull field name like `_.get(styleOptions, 'field.name')`? Do you want to change them all in this PR since you changed this one?

```
Users/nathanreese/projects/kibana/x-pack/legacy/plugins/maps/public/layers/styles/vector/components/label/dynamic_label_form.js:
   28          <FieldSelect
   29            fields={fields}
   30:           selectedFieldName={_.get(styleOptions, 'field.name')}
   31            onChange={onFieldChange}
   32            compressed

/Users/nathanreese/projects/kibana/x-pack/legacy/plugins/maps/public/layers/styles/vector/components/orientation/dynamic_orientation_form.js:
   31          <FieldSelect
   32            fields={fields}
   33:           selectedFieldName={_.get(styleOptions, 'field.name')}
   34            onChange={onFieldChange}
   35            compressed

/Users/nathanreese/projects/kibana/x-pack/legacy/plugins/maps/public/layers/styles/vector/components/size/dynamic_size_form.js:
   51            <FieldSelect
   52              fields={fields}
   53:             selectedFieldName={_.get(styleOptions, 'field.name')}
   54              onChange={onFieldChange}
   55              compressed

/Users/nathanreese/projects/kibana/x-pack/legacy/plugins/maps/public/layers/styles/vector/components/symbol/dynamic_icon_form.js:
   50            <FieldSelect
   51              fields={fields}
   52:             selectedFieldName={_.get(styleOptions, 'field.name')}
   53              onChange={onFieldChange}
   54              compressed
```